Readme.md

```diff
@@ -1,213 +1,108 @@
+* *Commit-graph:* accelerates commit walks and reachability calculations,
+   speeding up commands like `git log`.
 
-### Installing on macOS
+* *Multi-pack-index:* enables fast object lookups across many pack-files.
 
-Scalar is installed and updated on macOS via [Homebrew](https://brew.sh/).
+* *Incremental repack:* Repacks the packed Git data into fewer pack-file
+  without disrupting concurrent commands by using the multi-pack-index.
 
-```sh
-brew tap microsoft/git
-brew install --cask scalar
-```
+As new versions of Git are releases, we update the list of features that
+Scalar automatically configures. This reduces your effort to keep your
+repositories as efficient as possible.
 
-If you wish to use the [GVFS Protocol][gvfs-protocol], then you will
-instead need the cask that uses [our custom fork of Git][microsoft-git]:
+## Scalar has moved!
 
-```sh
-brew tap microsoft/git
-brew install --cask scalar-azrepos
-```
+Through significant effort from our team, we have successfully transitioned
+Scalar from a modified version of [VFS for Git][vfsforgit] into a thin shell
+around core Git features. The Scalar executable has now been ported to be
+included [in the `microsoft/git` fork][microsoft-git]. Please visit that
+fork for all of your Scalar needs:
 
-When new versions of Scalar are available, you can upgrade in a few
-different ways. First, you can use `brew`:
+* Download [the latest `microsoft/git` release][releases].
+* Read [the Scalar documentation][docs].
+* Contribute changes [to the `scalar` CLI][scalar-cli].
 
-```sh
-brew update
+### Why did Scalar move?
 
-# Use only one of the following, depending on which you have installed:
-brew upgrade --cask scalar
-brew upgrade --cask scalar-azrepos
-```
+Scalar started as a modification of [VFS for Git][vfsforgit] in order to
+create a working solution with a robust test suite in a short amount of
+time. The goal was to depend more on features that exist within Git itself
+instead of creating new functionality within this project. Since the start,
+we have focused on this goal with efforts such as
+[improving sparse-checkout performance in Git][sparse-checkout-blog],
+[implementing background maintenance in Git][background-maintenance],
+and [integrating the GVFS protocol into `microsoft/git`][remove-read-object]
+which allowed us to [drop the `Scalar.Mount` process][remove-mount].
+All of these changes reduced the size of the code in Scalar itself until
+it could be replaced [with a small command-line interface][scalar-cli].
 
-Alternatively, you can run `scalar upgrade` and it will run the necessary
-`brew` commands on your behalf.
+[sparse-checkout-blog]: https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/
+[background-maintenance]: https://github.blog/2021-03-15-highlights-from-git-2-31/#introducing-git-maintenance
+[remove-read-object]: https://github.com/microsoft/scalar/pull/122
+[remove-mount]: https://github.com/microsoft/scalar/pull/222
```

Comment:
I may be going crazy, but I'm not actually seeing these in preview?

Reply:
These links are referenced by links further up in the document, allowing me to keep the long URLs out of the paragraphs.